### PR TITLE
update Genome to use appropriate observer APIs

### PR DESCRIPTION
**Pull request by nnutter**

### CODE REVIEW

**Comment by nnutter** [comment]
I'm going to merge this, without review, to fix master.  *Please* audit this and post feedback on this PR if you believe changes need to or should be made.

File: lib/perl/Genome/Site/TGI/Extension/Sys/Lock.pm [hunk 1]
```diff
@@ -41,7 +41,8 @@ if ($nessy_server) {
     );
     Genome::Sys::Lock->add_backend('site', $nessy_site_backend);
 
-    UR::Context->process->add_observer(
+    UR::Observer->register_callback(
+        subject => UR::Context->process,
```
Comment (nnutter) [inline lib/perl/Genome/Site/TGI/Extension/Sys/Lock.pm]: genome/genome#753 fixed this call.  `register_callback` does not support `subject`; `subject_id` and `subject_class_name` must be supplied.

File: lib/perl/Genome/Config/AnalysisProject.pm [hunk 2]
```diff
@@ -81,8 +81,8 @@ class Genome::Config::AnalysisProject {
     id_generator => '-uuid',
 };
 
-__PACKAGE__->add_observer(aspect => 'is_cle', callback => \&_is_updated);
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject.pm]: Is it now wrong to call `add_observer` as a class method?

Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject.pm]: Nevermind. I should read your commit messages! :smile:

**Comment by ebelter** [comment]
+1

**Pull request merged**